### DISCUSSION TITLE
Enable "Slow-Mode" Option for 0.5x Experience?

### DISCUSSION THREAD

**Discussion by MasterPooBaa** [chromiecraft/chromiecraft#5122] [Suggestions]
I was wondering if the Devs could implement/enable a 0.5x experience option?

I'm sure theres at least a few of us out there that just enjoy the journey, doing all the quests in an area and not feeling like one is steamrolling ones way to 80.  Not asking for any special title/reward etc.

Effort to implement should be low - we allready have exp on/off feature and the far more complicated .weekend exp rate system.

Thank you in advance - even if you just spend time reading this :o)

Regards

**Comment by MasterPooBaa** [comment]
No update on this? :(

**Comment by Vyperhand** [comment]
This is a challenge on TWoW, actually.  Would need a different name as I think they call it ironman there.  But maybe make this a fourth challenge option with fewer restrictions?  Can still use AH, etc?

**Reply by MasterPooBaa** [reply]
I've done both IronMan, IronMan Bloodthirsty and HardCore.
Ironman is *technically* more difficult as all of your gear has zero bonus stats, but that also means there is little to no gear choice, quests dont matter (beyond the exp), and you dont get the choice/fun of playing with professions.  AH and GB access with Ironman is of little use also, besides getting bigger bags early or the odd bit of gear you just cant get from drops.
So HardCore + 0.5x would be awesome!
*Thinks*  Turtle wow had a similar challenge - Hardcore + optional scrolls you could use that enabled "Slow and Steady" (0.5x exp) and "Exhaustion" (No rested exp).   That was of course for a vanilla 1.13 client tho.

**Reply by ghost** [reply]
> I've done both IronMan, IronMan Bloodthirsty and HardCore. Ironman is _technically_ more difficult as all of your gear has zero bonus stats, but that also means there is little to no gear choice, quests dont matter (beyond the exp), and you dont get the choice/fun of playing with professions. AH and GB access with Ironman is of little use also, besides getting bigger bags early or the odd bit of gear you just cant get from drops. So HardCore + 0.5x would be awesome! _Thinks_ Turtle wow had a similar challenge - Hardcore + optional scrolls you could use that enabled "Slow and Steady" (0.5x exp) and "Exhaustion" (No rested exp). That was of course for a vanilla 1.13 client tho.

Not sure this is the right place to ask, but what was your experience doing Bloodthirsty like? Thinking of rolling a Bloodthirty Warrior!

**Reply by MasterPooBaa** [reply]
I consider BT easier than regular IM. As it essentially just farming, you find a nice safe spot and start grinding.  No need to go into dangerous caves, avoid roving elites, avoid janky quest mechanics etc.  You just have to like farming :)

**Comment by ghost** [comment]
Hoping this gets some love, 0.5x would be so much fun!

Edit: also, 0.5x might help with quest bug triage!

**Reply by andovcr** [reply]
@OAguinagalde can confirm, but I believe their work on this PR: [azerothcore/mod-weekend-xp/pull/30](https://github.com/azerothcore/mod-weekend-xp/pull/30) would allow for this, and is set to be merged into AC shortly.

**Comment by OAguinagalde** [comment]
That is right, my PR changes the mod so that rates so that `floats` can be used and changed the previously hardcoded minimum rate value of 1to 0, but I'm not sure about it being merged soon though. I got it implemented but as far as I know no one reviewed it yet, but hopefully it happens soon.

**Reply by ghost** [reply]
Thank you for working on this!

**Reply by MasterPooBaa** [reply]
Ditto!  Thanks for working on this!
Are there any plans to institute sub-1x for Hardcore/Ironman/Bloodthirsty modes?
Currently they appear locked at 1x.

**Reply by ghost** [reply]
Wow, I feel like sub-1x hardcore should have its own name! Super hardcore?

**Reply by MasterPooBaa** [reply]
Slowcore! haha
